src/display/canvas.js

```diff
@@ -706,7 +706,8 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
   CanvasGraphics.prototype = {
 
     beginDrawing: function CanvasGraphics_beginDrawing(transform, viewport,
```

Comment:
Not sure this can be done as the transparency argument doesn't seem to come from params:

```js
      this.gfx.beginDrawing(params.transform,
                            params.viewport,
                            transparency,
                            params.backgroundColor);
```